WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -414,8 +381,6 @@ private extension AddEditCoupon {
         static let addDescriptionPlaceholder = NSLocalizedString("Add the description of the coupon.",
                                                                  comment: "Placeholder text that will be shown in the view" +
                                                                  " for adding the description of a coupon.")
-        static let expiryDateActionSheetTitle = NSLocalizedString("Set an expiry date for this coupon",
-                                                                  comment: "Title of the action sheet for setting an expiry date for a coupon.")
         static let actionSheetEditExpirationDate = NSLocalizedString("Edit expiration date",
                                                                      comment: "Button in the action sheet for editing the expiration date of a coupon.")
         static let actionSheetDeleteExpirationDate = NSLocalizedString("Delete expiration date",
```

Comment:
I missed this earlier, but we should remove these also, right?

Reply:
Thanks for catching this one, I'm still struggling to find a reliable code analysis in Xcode to point out introduced problems in my changelog, so I was looking for unused strings resources manually 😬 Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/6953/commits/f103f82b572f72922ba21c2014bb69e7eb073e53

Reply:
I don't know of a good tool for that either 😅 I'll see if I can find anything!